### ISSUE TITLE
Invalid argument supplied for foreach()

### ISSUE THREAD

**Issue by knynkwl** [engram-design/Workflow#117]
### Description
I recently updated and now content editors are getting this error when trying to submit for approval on pages.

```
    Invalid argument supplied for foreach()

     * Returns the reviewer user groups for the given submission.
     *
     * @param Submission|null $submission
     * @return UserGroup[]
     */
    public function getReviewerUserGroups(Submission $submission = null): array
    {
        $userGroups = [];
 
        foreach (Workflow::$plugin->getSettings()->reviewerUserGroups as $reviewerUserGroup) {
```

### Additional info

- Plugin version: 1.5.1
- Craft version: 3.4.22.1

**Comment by knynkwl** [comment]
I looks like the "Reviewer User Groups" field needed to have a user group selected. It's not a required field though, is this the correct behavior?

**Comment by engram-design** [comment]
Reviewer User Groups are certainly optional, so that's quite odd behaviour. I also can't replicate this on my end. The default of `reviewerUserGroups` should be an empty array, but for your case, it looks like this might be null - somehow.

Are you using a custom `workflow.php` config file in your `/config` folder?

**Comment by engram-design** [comment]
Whatever the case, this should be fixed in [1.5.2](https://github.com/verbb/workflow/releases/tag/1.5.2)
